### PR TITLE
implemented csv parser

### PR DESCRIPTION
**Pull request by Nick97-git**

### CODE REVIEW

File: src/main/java/com/amazon/review/util/ParserCsv.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+@Component
+public class ParserCsv {
+
+    @SneakyThrows
+    public void parseCsv(List<String> lines) {
+        CsvParserSettings settings = new CsvParserSettings();
+        settings.setMaxCharsPerColumn(100000);
+        CsvParser csvParser = new CsvParser(settings);
+        long s = System.currentTimeMillis();
```
Comment (boroda4436) [inline src/main/java/com/amazon/review/util/ParserCsv.java:20]: bad variable naming 's'

File: src/main/java/com/amazon/review/util/ParserCsv.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import java.time.ZoneId;
+import java.util.List;
+import lombok.SneakyThrows;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ParserCsv {
+
+    @SneakyThrows
+    public void parseCsv(List<String> lines) {
```
Comment (boroda4436) [inline src/main/java/com/amazon/review/util/ParserCsv.java:16]: how would you test this void method?

File: src/main/java/com/amazon/review/util/ParserCsv.java (hunk 1 above)
Comment (Nick97-git) [reply src/main/java/com/amazon/review/util/ParserCsv.java:20]: made speed test and forgot to delete)

File: src/main/java/com/amazon/review/util/ParserCsv.java (hunk 2 above)
Comment (Nick97-git) [reply src/main/java/com/amazon/review/util/ParserCsv.java]: I don't know if i'm right but the only option here is  to change return type to boolean and test if true comes or not

File: src/main/java/com/amazon/review/util/ParserCsv.java [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        for (int i = 1; i < lines.size(); i++) {
+            String[] data = csvParser.parseLine(lines.get(i));
+            String productId = data[1];
+            String userId = data[2];
+            String profileName = data[3];
+            long helpfulnessNumerator = Long.parseLong(data[4]);
+            long helpfulnessDenominator = Long.parseLong(data[5]);
+            long score = Long.parseLong(data[6]);
+            LocalDateTime time = convertToLocalDateTime(data[7]);
+            String summary = data[8];
```
Comment (boroda4436) [inline src/main/java/com/amazon/review/util/ParserCsv.java:27]: Where are you using this property?
The boolean return type does not make any sense. Instead, return the list of some objects based on the input value.

**Pull request merged**